src/pages/api/profile/get-address.ts

```diff
@@ -0,0 +1,115 @@
+import { getServerSideCookies, Endpoint, HttpStatus } from "@fwl/web";
+import {
+  loggingMiddleware,
+  wrapMiddlewares,
+  tracingMiddleware,
+  errorMiddleware,
+  recoveryMiddleware,
+  rateLimitingMiddleware,
+} from "@fwl/web/dist/middlewares";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { Handler } from "@src/@types/handler";
+import { UserCookie } from "@src/@types/user-cookie";
+import { configVariables } from "@src/configs/config-variables";
+import { logger } from "@src/configs/logger";
+import { initProfileClient } from "@src/configs/profile-client";
+import getTracer from "@src/configs/tracer";
+import { ERRORS_DATA, webErrorFactory } from "@src/errors/web-errors";
+import { authMiddleware } from "@src/middlewares/auth-middleware";
+import { sentryMiddleware } from "@src/middlewares/sentry-middleware";
+import { getProfileAccessToken } from "@src/utils/get-profile-access-token";
+
+const handler: Handler = async (request, response) => {
+  const webErrors = {
+    invalidProfileToken: ERRORS_DATA.INVALID_PROFILE_TOKEN,
+    invalidScopes: ERRORS_DATA.INVALID_SCOPES,
+    profileDataNotFound: ERRORS_DATA.PROFILE_DATA_NOT_FOUND,
+    unreachable: ERRORS_DATA.UNREACHABLE,
+    invalidQueryString: ERRORS_DATA.INVALID_QUERY_STRING,
+    addressNotFound: ERRORS_DATA.ADDRESS_NOT_FOUND,
+  };
+
+  return getTracer().span("get-address handler", async (span) => {
+    const userCookie = await getServerSideCookies<UserCookie>(request, {
+      cookieName: "user-cookie",
+      isCookieSealed: true,
+      cookieSalt: configVariables.cookieSalt,
+    });
+
+    if (!userCookie) {
+      response.statusCode = HttpStatus.TEMPORARY_REDIRECT;
+      response.setHeader("location", "/");
+      response.end();
+      return;
+    }
+
+    const addressId = request.query.addressId;
+
+    if (!addressId || typeof addressId !== "string") {
+      throw webErrorFactory(webErrors.invalidQueryString);
+    }
+
+    const profileAccessToken = await getProfileAccessToken(
+      userCookie.access_token,
+    );
+    span.setDisclosedAttribute(
+      "is Connect.Profile access token available",
+      true,
+    );
```

Comment:
```suggestion
    span.setDisclosedAttribute(
      "Connect.Profile access token",
      profileAccessToken,
    );
```
So if there's any error after this line, we can get the token from tracers and use it to make some tests

Reply:
Isn't access token sensitive data ? 
Shouldn't it be `span.setAttribute` if we decide to put it in the span ?

Reply:
I don't think so, what we have inside is not private, juste the sub and scopes and the signed part is "only" there to prove it's a valid token